src/status_im/subs/wallet/activities.cljs

```diff
@@ -115,13 +115,13 @@
                                              (comp :network-name first))
          address-activities     (->> (get activities current-viewing-account-address)
                                      (vals)
-                                     (sort :timestamp))]
+                                     (sort-by :timestamp))]
      (->> address-activities
           (keep #(process-activity-by-type chain-id->network-name %))
           (group-by (fn [{:keys [timestamp]}]
                       (datetime/timestamp->relative-short-date (* timestamp 1000))))
```

Comment:
It's probably fine, but just so you know, this is a _time bomb_ because the relative date depends on the concept of now and on the timezone. Here you can check how we use `with-redefs` to force the dynamic var `cljs-time.core/*ms-fn*` to point to a stable now.

https://github.com/status-im/status-mobile/blob/4b7c906df27fd3bdf90dd80048cff5f87b139514/src/utils/datetime_test.cljs#L145-L149

I remember seeing this type of problem in our pipeline and in general in my experience.

----

On a tangent, in some sense, one could say the subscription is stateful because the subscription output can vary based on the current time. In re-frame, for example, we inject the current timestamp via an interceptor to avoid this:

https://github.com/status-im/status-mobile/blob/4b7c906df27fd3bdf90dd80048cff5f87b139514/src/status_im/setup/interceptors.cljs#L12

But it's okay as is, just felt like mentioning it's not ideal.